pkgs/development/python-modules/aws-sam-translator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, boto3
+, enum34
+, jsonschema
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "aws-sam-translator";
+  version = "1.5.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9d8a25e058c78d2cef5c07aec7f98cbc2070dbfc2eb6a2e102a16beafd14e3ca";
+  };
+
+  # Tests are not included in the PyPI package
```

Comment:
You may want to use `fetchFromGitHub` in that case.

Reply:
I looked into it but the versions don't line up, latest tag on GitHub is 1.5 but latest version on PyPi is 1.5.4. The released version is probably just master but they really should tag it.

Reply:
Well, then better stick with `fetchPypi`.